### PR TITLE
(Graphviz) Shimmed all binaries in Graphviz package

### PR DESCRIPTION
**Pull request by alexbrinister**
## Description
Added shims for all binaries in the Graphviz package. Graphviz provides binaries other than `dot` that are sometimes necessary. Providing access to all shipped utilities is also in line with how many Linux distributions package this software.

## Motivation and Context
Expose all utilities shipped by Graphviz without needing to specifying the full directory to the utilities or putting the install location in Program Files on that `PATH`. For example, I had an immediate need for `unflatten.exe` but this is not shimmed by the latest version of the package. To work around this issue right now, I have to add the install location to the `PATH` (or specify the full paths manually), which obviates the need to shim `dot.exe`.

Note that this cannot be fixed simply by using the `dot` executable with an argument; some of the utilities (such as `unflatten`) are separate applications.

## How Has this Been Tested?
`choco install graphviz --debug --verbose --source <workdir>`. No errors, all files (and shims) in place.

`choco uninstall graphviz --debug --verbose --source <workdir>`. No errors and all files (and shims) removed.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/graphviz/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -16,7 +16,10 @@ Remove-Item $toolsPath\*.exe -ea 0
 
 $packageName = $packageArgs.packageName
 $installLocation = Get-AppInstallLocation $packageArgs.softwareName
-if (!$installLocation)  { Write-Warning "Can't find $packageName install location"; return }
+if (!$installLocation)  { Write-Warning "Can't find $packageName install location"; return 1 }
```
Comment (AdmiringWorm) [inline automatic/graphviz/tools/chocolateyInstall.ps1:19]: With this change you may have broken the package.
The package may fail to install when you return `1` in this context, even though the software have already been installed.
Please revert this change.

File: automatic/graphviz/tools/chocolateyUninstall.ps1 [hunk 2]
```diff
@@ -12,7 +12,22 @@ if ($key.Count -eq 1) {
...
             validExitCodes         = @(0)
             file                   = "$($_.UninstallString.Replace(' /x86=0', ''))"   #"C:\Program Files\OpenSSH\uninstall.exe" /x86=0
+            softwareName           = 'Graphviz*'
         }
+
+        $installLocation = Get-AppInstallLocation $packageArgs.softwareName
+        Write-Debug "$packageName installed in: $installLocation"
+        
+        if (!$installLocation) { 
+            Write-Warning "Can't find $packageName install location"; return 1
```
Comment (AdmiringWorm) [inline automatic/graphviz/tools/chocolateyUninstall.ps1:22]: As with previous comment, please don't return `1` here.
Just return, without any exit code.

File: automatic/graphviz/tools/chocolateyUninstall.ps1 [hunk 3]
```diff
@@ -12,7 +12,22 @@ if ($key.Count -eq 1) {
             fileType               = 'EXE'
             validExitCodes         = @(0)
             file                   = "$($_.UninstallString.Replace(' /x86=0', ''))"   #"C:\Program Files\OpenSSH\uninstall.exe" /x86=0
+            softwareName           = 'Graphviz*'
         }
+
+        $installLocation = Get-AppInstallLocation $packageArgs.softwareName
```
Comment (AdmiringWorm) [inline automatic/graphviz/tools/chocolateyUninstall.ps1:18]: At this point, you should not need to use the `Get-AppInstallLocation` helper, you can query the registry directly by using the `$key` variable (if possible).

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]

File: automatic/graphviz/tools/chocolateyInstall.ps1 (hunk 1 above)
Comment (alexbrinister) [reply automatic/graphviz/tools/chocolateyInstall.ps1]: Fixed. Thank you for the feedback!

File: automatic/graphviz/tools/chocolateyUninstall.ps1 (hunk 2 above)
Comment (alexbrinister) [reply automatic/graphviz/tools/chocolateyUninstall.ps1]: Fixed. Thank you for the feedback!

File: automatic/graphviz/tools/chocolateyUninstall.ps1 (hunk 3 above)
Comment (alexbrinister) [reply automatic/graphviz/tools/chocolateyUninstall.ps1]: Fixed. Thank you for the feedback!

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Comment by AdmiringWorm** [comment]
@alexbrinister your changes have been merged, thanks for your contribution 👍

A new fix version will be pushed to chocolatey.org.
